src/opm/output/eclipse/AggregateConnectionData.cpp

```diff
@@ -86,28 +63,27 @@ namespace {
              wellID < nWell; ++wellID)
         {
             const auto& well = wells[wellID];
-            const auto& conn0 = well.getConnections();
-            const auto& conns = Opm::WellConnections( conn0, grid );
-            const int niSI = static_cast<int>(conn0.size());
-            std::map <std::size_t, const Opm::Connection*> sIToConn;
+            std::vector<const Opm::Connection*> connSI;
+            for (const auto& conn : well.getConnections()) {
+                if (grid.cellActive(conn.getI(), conn.getJ(), conn.getK()))
+                    connSI.push_back( std::addressof(conn) );
```

Comment:
There's no benefit to using `addressof` here (besides, you need to `#include <memory>` for that library function too).  Just use the `&` operator.

Reply:
Well - personally I think `std::addressof()` is much more readable; I will add the include though.

Reply:
> Well - personally I think `std::addressof()` is much more readable

Except you're also communicating to the reader that even though they are in a context with perfect knowledge about all types involved they're not allowed to use that knowledge and must resort to treating everything as bags of bits and that `reinterpret_cast<>` is appropriate.